### ISSUE TITLE
T4-S3 Display no powering display from battery

### ISSUE THREAD

**Issue by klone52** [Xinyuan-LilyGO/LilyGo-AMOLED-Series#72]
Hi , I'm having an issue where using any sketch I see the display powered On and everything is ok if I plug the USB before , and stay On when unplugged with the battery connected. But if I only use the battery, the pcb is  powered but the display doesn't turn on.

**Comment by lewisxhe** [comment]
I need to check the PCB and then tell you the test method. I will reply you tomorrow.

**Comment by lewisxhe** [comment]
Connect the battery and measure whether there is battery voltage at these two ends

![WXWorkCapture_17325022315755](https://github.com/user-attachments/assets/88f1d92c-f1d7-470d-a99f-336dc1a3ffc2)

**Comment by klone52** [comment]
Hi ! with my battery on 3.8V i see 2.49V on this point

**Comment by lewisxhe** [comment]
OK, I suspect that this MOS is damaged. Connect D and S according to the instructions in the figure below and see if the power supply is normal.
![image](https://github.com/user-attachments/assets/0163882e-47b8-44c6-95ca-6ad640b3e6cb)

**Comment by dobodu** [comment]
Hi lewis,

My T4-S3 has lost display that was working well a few days ago. I power-it up with USB.
Would you give me an advice if I need to check anything on the mainboard ?
The display itself seems to be working (without light) since when I draw lines, I have no error and the
correct time to proceed. But no pixel enlighten...

**Comment by lewisxhe** [comment]
There is a high possibility that the screen is damaged, and the screen power supply voltage needs to be tested. This test point must be tested by removing the motherboard, and the screen must be connected during the test.
![image](https://github.com/user-attachments/assets/38547024-caff-47b7-9bf4-0acb95b68b21)

**Comment by dobodu** [comment]
Thank you Lewis, I'll try the test point.
I hope T3 S4 with enclosure is not hard to dissassembly.

**Comment by dobodu** [comment]
Hi Lewis,
I've been checkin the testpoint. A expected ELVSS = -3.6V and ELVDD = +3.6V.
Checking the schematics, I think EL6804 is powering correctly the OLED display which is dead i believe.
Am I right ?

**Comment by lewisxhe** [comment]
ELVSS and ELVDD are in the normal voltage range. If the screen doesn't light up, the only possibility is that the screen is broken, or maybe the motherboard. Have you tested the factory firmware?

**Comment by dobodu** [comment]
Yes, uploaded factory firmware, board is getting on load but no display...
